### PR TITLE
Add totalCountTableReducer and modify song table layout.

### PR DESCRIPTION
**Pull request by hougs**
Modify table layout to include a column for the total count, as well as only keep one version in the default locality group of the song table.

Also adds a reducer that writes the total plays to a command line specified column.

### CODE REVIEW

**Comment by hougs** [comment]
@kryzthov A reducer and some layout changes for you!

File: src/main/java/org/kiji/music/reduce/TotalCountTableReducer.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import org.kiji.mapreduce.KijiTableContext;
+import org.kiji.mapreduce.KijiTableReducer;
+
+/**
+ * A reducer that will sum the values associated with a key and write that sum
+ * to a column specified via command line flags.
+ */
+public class TotalCountTableReducer extends KijiTableReducer<Text, LongWritable> {
+  // For our music project, we are writing to column family "derived".
+  @Flag(name="column-family", usage="Specify the column family to write to.")
```
Comment (kryzthov) [inline src/main/java/org/kiji/music/reduce/TotalCountTableReducer.java:37]: Hmmm, how are these flags initialized?
I guess they should be stored in the job Configuration somehow?

Comment (hougs) [inline src/main/java/org/kiji/music/reduce/TotalCountTableReducer.java:37]: This should just work. At least, it has in the past and I think a change in that would be a regression.

Comment (kryzthov) [inline src/main/java/org/kiji/music/reduce/TotalCountTableReducer.java:37]: Okay, I have no idea how this works if it does, but I'll trust you for now :)

**Comment by kryzthov** [comment]
29982e6e5ae32de839c5a8ff383c45c75e3fc3f8

**Pull request closed without merging**